website/src/pages/index.jsx

```diff
@@ -121,13 +124,20 @@ const Steroids = () => {
               title="GET STARTED"
               link="/docs/usage/getting-started/"
             />
-            {/* <div sx={{ color: 'gray4', mt: 4 }}>
+            <div sx={{ color: 'gray4', mt: 4 }}>
               Trusted by the following companies and projects:
             </div>
             <div sx={{ mt: 3 }}>
-              <Soundcloud sx={{ height: 30 }} />
-              <Mytaxi sx={{ ml: 3, height: 30 }} />
-            </div> */}
+              <a href="https://soundcloud.com" target="__blank">
```

Comment:
do you think you should officially ask these companies to have their logo on the website?